addons/purchase/static/src/product_catalog/kanban_record.js

```diff
@@ -31,7 +32,7 @@ patch(ProductCatalogKanbanRecord.prototype, {
 
     async updatePackagingQuantity(packaging) {
         const productPackagingQty =
-            Math.floor(this.productCatalogData.quantity / packaging.qty) + 1;
+            Math.floor(parseFloat(formatFloat(this.productCatalogData.quantity / packaging.qty))) + 1;
```

Comment:
You're going to have rounding error issues if you don't pass the `precision`, i.e. `formatFloat` defaults to 2 digits whereas the packaging qty follows the `'Product Unit of Measure'` precision. You may be able to leverage the `this.env.precision` though since `quantity` uses the same precision. I didn't check though.

Reply:
I did the following example in your runbot:

https://github.com/user-attachments/assets/0edf5c8a-5b1a-4092-b71d-d11c75fc5924

There are 2 rounding issues, the first is the wrong quantity is added for the product, the second is that the rounding error gets added together the more you click on the add package button (i.e. 22.6812 is rounded to 22.69 so clicking twice = 45.38 instead of 45.2624)